language/German/strings.po

```diff
@@ -884,21 +885,26 @@ msgstr "WAN"
 
 #: src/frontend/mame/ui/info.cpp:70
 msgid ""
-"One or more ROMs/disk images for this system are incorrect. The system may "
-"not run correctly.\n"
+"One or more ROMs/disk images for this system are incorrect. The system may not "
+"run correctly.\n"
 msgstr ""
+"Ein oder mehrere ROMs/Disk-Images für dieses System sind inkorrekt. Das System "
+"läuft möglicherweise nicht richtig.\n"
 
 #: src/frontend/mame/ui/info.cpp:83
 msgid ""
 "There are known problems with this system\n"
 "\n"
 msgstr ""
+"Es gibt bekannte Probleme mit diesem System\n"
+"\n"
 
 #: src/frontend/mame/ui/info.cpp:88
 msgid ""
-"One or more ROMs/disk images for this system have not been correctly "
-"dumped.\n"
+"One or more ROMs/disk images for this system have not been correctly dumped.\n"
```

Comment:
Please back out this rewrapped English message, too

Reply:
> Is that the terminology the emulation community uses, though?

true, but if we're translating "solid state media", "screenshot" and other well-known terms into German, "dumping" should also be translated? Then again I'm not sure who the target audience is for the German translation, i.e. what can you assume a random user knows and doesn't know. If I had done the translation I would have left many more english terms as-is

Reply:
Fixed

Reply:
> true, but if we're translating "solid state media", "screenshot" and other well-known terms into German, "dumping" should also be translated? Then again I'm not sure who the target audience is for the German translation

The target audience is the German emulation community.  The translation should use idiomatic language that a typical German emulator user would use when writing.

Reply:
"gedumpt" is fine in this case then I guess...